backend/src/rest/assignments.js

```diff
@@ -14,8 +14,12 @@ router.post('/', async (req, res) => {
   let title = req.body.title
   let description = req.body.description
   let contact = req.body.contact
+  let customerFee = req.body.customerFee
+  let customerCompanyURL = req.body.customerCompanyURL
+  let clientHourlyRate = req.body.clientHourlyRate
+  let location = req.body.location
```

Comment:
Behöver denna indata valideras? Vad händer om man skickar in en lista eller ett objekt?